src/coreclr/src/vm/dispatchinfo.cpp

```diff
@@ -1602,7 +1602,7 @@ void DispatchInfo::InvokeMemberWorker(DispatchMemberInfo*   pDispMemberInfo,
                         COMPlusThrowHR(DISP_E_BADPARAMCOUNT);
 
                     // Retrieve the method descriptor that will be called on.
-                    MethodDesc *pMD = GetFieldInfoMD(METHOD__FIELD__GET_VALUE, pObjs->MemberInfo->GetTypeHandle());
+                    MethodDesc *pMD = GetFieldInfoMD(METHOD__FIELD_INFO__GET_VALUE, pObjs->MemberInfo->GetTypeHandle());
```

Comment:
What was the problem that this change is fixing?

Reply:
The new linker was warning that it can't find `GetValue` and `SetValue` methods on `RuntimeFieldInfo`.

It was looking for them there because we generate a linker XML file based on `corelib.h` here - 

https://github.com/dotnet/runtime/blob/b552749906402ec7994150bc9dd8970440dbb9c3/src/coreclr/src/System.Private.CoreLib/CreateRuntimeRootILLinkDescriptorFile.targets#L25-L44

Since those methods exist on the base `FieldInfo` class, I moved coreclr's defines to the correct class.